content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,141 @@ Use VoIP services in Odoo with OnSIP
 
 Configuration
 =============
 
-Go to Apps and install the module **VoIP OnSIP**.
+First, go to Apps and install the module *VoIP OnSIP*.
+
+.. tip::
+   Clear out the search bar and type in `onsip`. The OnSIP module will appear.
 
-.. image:: onsip/onsip01.png
+.. image:: onsip/install-onsip.png
    :align: center
+   :alt: View of Onsip app in the app search results.
+
+Odoo VoIP setting
+-----------------
 
-Go to Settings/General Settings. In the section Integrations/Asterisk (VoIP), fill in the 3 fields:
+Following the module installation, go to :menuselection:`Settings --> General Settings`. In the
+:menuselection:`Integrations -- > Asterisk (VoIP)` section, fill in the 3 fields:
 
-- **OnSIP Domain** is the domain you chose when creating an account on www.onsip.com. If you don't
-  know it, log in to https://admin.onsip.com/ and you will see it in the top right corner of the
-  screen.
-- **WebSocket** should contain wss://edge.sip.onsip.com
-- **Mode** should be Production
+- :guilabel:`OnSIP Domain` is the domain that was assigned when creating an account on
+  `OnSIP <https://www.onsip.com/>`_.
+- :guilabel:`WebSocket` = `wss://edge.sip.onsip.com`
+- :guilabel:`Mode` = :guilabel:`Production`
 
-.. image:: onsip/onsip02.png
+.. image:: onsip/asterisk-setting.png
    :align: center
+   :alt: VoIP configuration settings in Odoo Settings app.
+
+.. tip::
+   To access the OnSIP domain, navigate to `OnSIP <https://www.onsip.com/>`_ and log in. An
+   administrative dahsboard will appear. In the left menu, click on :guilabel:`Users` and then
+   select any user. By default the user will open on the :guilabel:`User Info` tab. Click on the
+   :guilabel:`Phone Settings` tab to reveal  OnSIP configuration credentials. In the first column,
+   last in the list of credentials, is :guilabel:`Domain`.
+
+   .. image:: onsip/domain-setting.png
+      :align: center
+      :alt: Domain setting revealed (highlighted) on administrative panel of OnSIP management
+            console.
 
-Go to **Settings/Users**. In the form view of each VoIP user, in the Preferences tab, fill in the
-section **PBX Configuration**:
+Odoo user setting
+-----------------
 
-- **SIP Login / Browser's Extension**: the OnSIP 'Username'
-- **OnSIP authorization User**: the OnSIP 'Auth Username'
-- **SIP Password**: the OnSIP 'SIP Password'
-- **Handset Extension**: the OnSIP 'Extension'
+Next, the user will need to be setup in Odoo. Every user that will be associated with an OnSIP User
+must also be configured in the Odoo user's settings/preferences.
 
-You can find all this information by logging in at https://admin.onsip.com/users, then select the
-user you want to configure and refer to the fields as pictured below.
+First, navigate to :menuselection:`Settings --> Manage Users --> Select the User`. Once on the user
+that the OnSIP account should be configured with, click on the :guilabel:`Preferences` tab and
```

Comment:
Since this is 15, I would add that you have to click "Edit" before you can make changes to the record. Not sure if you want to add it before or after you mention scrolling to the VoIP section, but I wanted to call it out